components/scripts/lib/cli-parsers/gradle/02-cli-parser.m4

```diff
@@ -3,6 +3,7 @@
 # Created by argbash-init v2.10.0
 # ARG_HELP([This function is overridden later on.])
 # ARG_VERSION([print_version],[v],[version],[])
+# ARG_OPTIONAL_BOOLEAN([fail-if-not-optimized],[x],[])
```

Comment:
✅ Done.

I removed it from this PR. I haven't created another PR yet as I don't want to try to keep it up-to-date with this one. I'll create the new PR once this one is merged.  It can be easily added back by:

```
git revert 1f5d854b84afd8be24a3d74b29ae926b933fec6f
```